cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala

```diff
@@ -70,36 +72,38 @@ class FulltextIndexTest extends DocumentingTest {
           #In contrast to <<administration-indexes-introduction, btree indexes>>, a full-text index""".stripMargin('#'))
       p(
         """* can be applied to more than one label.
-          #* can be applied to relationship types (one or more).
+          #* can be applied to more than one relationship type.
           #* can be applied to more than one property at a time (similar to a <<administration-indexes-create-a-composite-index-for-nodes, _composite index_>>) but with an important difference:
           #While a composite index applies only to entities that match the indexed label and _all_ of the indexed properties, full-text index will index entities that have at least one of the indexed labels or relationship types, and at least one of the indexed properties.""".stripMargin('#'))
       p("For information on how to configure full-text indexes, refer to <<operations-manual#index-configuration-fulltext, Operations Manual -> Indexes to support full-text search>>.")
     }
-    section("Procedures to manage full-text indexes", "administration-indexes-fulltext-search-manage") {
-      p("""Full-text indexes are managed through built-in procedures, see <<operations-manual#neo4j-procedures, Operations Manual -> Procedures>> for a complete reference.
+    section("Procedures and commands to manage full-text indexes", "administration-indexes-fulltext-search-manage") {
+      p("""Full-text indexes are managed through commands and used through built-in procedures, see <<operations-manual#neo4j-procedures, Operations Manual -> Procedures>> for a complete reference.
           #
-          #The procedures for managing full-text indexes are listed in the table below:""".stripMargin('#'))
+          #The commands and procedures for using full-text indexes are listed in the table below:""".stripMargin('#'))
```

Comment:
I would maybe remove the word `using` all together since it now sounds like both the commands and procedures are for using the fulltext indexes, while you already have described in the sentence above the that commands are for managing and the procedures for using